src/components/tx/modals/BatchExecuteModal/ReviewBatchExecute.tsx

```diff
@@ -91,6 +112,19 @@ const ReviewBatchExecute = ({ data, onSubmit }: { data: BatchExecuteData; onSubm
         </Typography>
         <DecodedTxs txs={txsWithDetails} />
 
+        {canRelay ? (
```

Comment:
In the safe creation relay PR, this is called `willRelay`. I would suggest keeping the naming consistent.